boa/src/bytecompiler.rs

```diff
@@ -343,17 +437,15 @@ impl<'b> ByteCompiler<'b> {
                 }
             } else if let Some(jump_info) = self.jump_info.last_mut() {
                 jump_info.breaks.append(&mut info.breaks);
+                jump_info.try_continues.append(&mut info.try_continues);
             }
         }
     }
 
     #[inline]
     fn compile_access<'a>(&mut self, node: &'a Node) -> Access<'a> {
```

Comment:
Clippy is saying that the `self` binding is not being used anymore here, does it make sense to keep it?